### ISSUE TITLE
[TorchInductor] `efficient_conv_bn_eval_graph_transform_decomposed` misses the `conv2d.padding` overload

### ISSUE THREAD

**Issue by miyorena** [pytorch/pytorch#196166]
### 🐛 Describe the bug

The decomposed Conv-BN fusion handles `aten.conv2d.default`, but does not handle the equivalent `aten.conv2d.padding` form with string padding.  The missing case uses the same `aten.batch_norm.default` pattern and the same Conv-BN folding rewrite.

## Minimal reproducer

```python
from __future__ import annotations

import torch
from torch._inductor import config
from torch._inductor.fx_passes.pre_grad import efficient_conv_bn_eval_pass
import torch._inductor.fx_passes.efficient_conv_bn_eval  # noqa: F401


class Model(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.weight = torch.nn.Parameter(torch.randn(4, 3, 3, 3))
        self.bias = torch.nn.Parameter(torch.randn(4))
        self.register_buffer("running_mean", torch.randn(4))
        self.register_buffer("running_var", torch.rand(4) + 1.0)

    def forward(self, x: torch.Tensor) -> torch.Tensor:
        conv = torch.ops.aten.conv2d.padding(
            x,
            self.weight,
            self.bias,
            [1, 1],
            "same",
            [1, 1],
            1,
        )
        return torch.ops.aten.batch_norm.default(
            conv,
            None,
            None,
            self.running_mean,
            self.running_var,
            False,
            0.1,
            1e-5,
            torch.backends.cudnn.enabled,
        )


torch.manual_seed(1703)
model = Model().eval()
x = torch.randn(2, 3, 16, 16)
gm = torch.fx.symbolic_trace(model)

with torch.no_grad():
    expected = gm(x)

with config.patch(
    {"efficient_conv_bn_eval_fx_passes": True, "freezing": False}
):
    efficient_conv_bn_eval_pass.apply(gm.graph)

gm.graph.lint()
gm.recompile()

with torch.no_grad():
    actual = gm(x)

torch.testing.assert_close(actual, expected)

targets = [n.target for n in gm.graph.nodes if n.op == "call_function"]
print("contains conv2d.padding:", torch.ops.aten.conv2d.padding in targets)
print("contains batch_norm.default:", torch.ops.aten.batch_norm.default in targets)
```

## Observed behavior

The pass leaves the original pattern unchanged:

```
aten.conv2d.padding -> aten.batch_norm.default
```

Output:

```
contains conv2d.padding: True
contains batch_norm.default: True
```

In contrast, the corresponding `aten.conv2d.default` form is handled by the decomposed Conv-BN fusion. 

### Versions

PyTorch: `e289091`

cc @chauhang @penguinwu @voznesenskym @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @kadeng @muchulee8 @amjames @aakhundov @coconutruben @jataylo

**Comment by yashraj4** [comment]
I'd like to take this one.

Based on my research, I suppose `supported_convs` in `efficient_conv_bn_eval_graph_transform_decomposed` (`torch/_inductor/fx_passes/efficient_conv_bn_eval.py`) includes the `.default` convolution overloads but not the `.padding` overloads used by string-padding convolutions such as `padding="same"`.

Specifically, the following are currently missing:

- `aten.conv1d.padding`
- `aten.conv2d.padding`
- `aten.conv3d.padding`

Because of this, the `input_fn is cls` check fails for these overloads and the optimization pass silently becomes a no-op, which matches the repro behavior.

Will open a PR with this change plus a test case covering the repro above.
